wallet/src/actors/worker/methods.rs

```diff
@@ -615,6 +615,8 @@ impl Worker {
             self.params.sync_address_batch_length,
             self.params.sync_address_batch_length,
         )?;
+        // Flag that a synchronization is in progress
+        wallet.apply_on_state(|mut state| state.synchronization.set_syncing(true))?;
```

Comment:
Shouldn't this check if that flag is already set to true, and in that case return some error?

Reply:
I didn't dare to change any existing behavior. I'm setting the `is_syncing` flag just the sake of completion, but as you saw it's not being used anywhere :shrug:

Reply:
Also, if this function returns using a `?` on some error, this flag will never be set to `false`.